src/neo4clj/convert.clj

```diff
@@ -40,7 +40,7 @@
 
 (defmethod neo4j->clj java.util.List
   [entity]
-  (into [] entity))
+  (into [] (map neo4j->clj entity)))
```

Comment:
If `walk/prewalk` is used (instead of ~~`postwalk`~~), then is should be possible to not need mapping this through here.